### PR TITLE
fix(featured): Introduce partial notice data model

### PR DESCRIPTION
**Pull request by lutangar**
> Depends on https://github.com/dis-moi/backend/pull/296

Fix  https://trello.com/c/IOThAQOA/565-jusqu%C3%A0-5-contributions-phares-%C3%A0-un-m%C3%AAme-profil

### CODE REVIEW

File: src/app/lmem/notice.ts [hunk 1]
```diff
@@ -3,6 +3,16 @@ import { Ratings } from './rating';
 import { Contributor, ContributorId, NewContributor } from './contributor';
 import { captureMessage } from '../utils/sentry';
 
+export interface NoticeItem {
```
Comment (JalilArfaoui) [inline src/app/lmem/notice.ts:6]: Shouldn’t we try to merge `NoticeItem` and `BaseNotice` to one unique model ?

Comment (lutangar) [reply src/app/lmem/notice.ts:6]: Definitely! 
But can't do that until Friday though.

Comment (lutangar) [reply src/app/lmem/notice.ts]: Could have a second look at this, I had to update the API as well to get something neat:
> https://github.com/dis-moi/backend/pull/298

File: src/app/lmem/contributor.ts [hunk 2]
```diff
@@ -11,16 +12,12 @@ interface Picture {
...
   sort?: number;
-  exampleMatchingUrl: string;
-  noticeId: number;
-  noticeUrl: string;
 }
 
 interface Contribution {
-  example: PinnedNotice; // @deprecated use `pinnedNotices` instead
-  starred: PinnedNotice; // @deprecated use `pinnedNotices` instead
+  example: NoticeItem;
```
Comment (JalilArfaoui) [inline src/app/lmem/contributor.ts:20]: Do we still use this property ? If not, we could remove it since it’s deprecated …

Comment (lutangar) [reply src/app/lmem/contributor.ts:20]: `example` it's not actually deprecated, I thought it was but it's not, but will certainly be in the future.

Comment (JalilArfaoui) [reply src/app/lmem/contributor.ts:20]: Where is it used ?

Comment (lutangar) [reply src/app/lmem/contributor.ts:20]: Don't remember at the moment, but I'll dig this and create an issue.
Could we merge this fix and discuss this later?

Comment (JalilArfaoui) [reply src/app/lmem/contributor.ts:20]: Definitely, no blocker !

**Review by JalilArfaoui — APPROVED** [review]

**Pull request merged**